warehouse/query-core/src/main/java/datawave/query/language/parser/lucene/AccumuloSyntaxParser.java

```diff
@@ -358,7 +358,7 @@ final public QueryNode Clause(CharSequence field) throws ParseException {
         if (boost != null) {
             float f = (float) 1.0;
             try {
-                f = Float.valueOf(boost.image).floatValue();
+                f = Float.parseFloat(boost.image);
```

Comment:
This spotbugs issue was found with the updated spotbugs plugin. However, it appears that this file is generated code that has been checked in to the repository. Is this file frequently re-generated and clobbered, or is it okay to make these fixes here? Would the code-generation utilities generate better (non-buggy) code if it were updated? Where is that code generator executed?